docs/authenticator_plugins.md

```diff
@@ -0,0 +1,80 @@
+# Authenticator Plugins
+
+django-ansible-base provides a plugable authentication system (see authentication.md for high level details) that are backed by individual plugin files. The plugin files are meant to provide access to specific authentication methods like SAML, ldap, OIDC, tacacs, etc. 
+
+This document outlines how to create authenticator plugins.
+
+## Creating an authenticator plugin
+
+### The File
+Authenticator plugins in django-ansible-base should be individual files in the ansible_base/authenticator_plugins folder. That being said, you can configure django-ansible-base to look in additional directories for the authenticator_plugins (see authentication.md). The file should be a logical name for the backend service you are communicating with; for example, saml.pl houses the code for connecting to SAML. 
```

Comment:
did you mean `saml.py` ?

Reply:
Oh you didn't hear? We're rewriting the entire project in Perl. 🤣 j/k